### DISCUSSION TITLE
Is there a way to access `JRELEASER_DRY_RUN` in `jreleaser.yml`?

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1433] [General]
I'm attempting to have Github's release set as `draft` only when JReleaser is doing a dry run.
Dry run is specified via the `JRELEASER_DRY_RUN` env variable.
Is there a way how to access env variables in general and `JRELEASER_DRY_RUN` in particular in `jreleaser.yml`?

**Comment by aalmiray** [comment]
You can try using a template value such as `{{ Env. JRELEASER_DRY_RUN }}`, as long as the target property supports templates. The guide shows an icon next to properties that do support templates.
